src/components/pages/user.yml

```diff
@@ -0,0 +1,3 @@
+accountClosed:
+  en: This user account is closed.
+  de: Dieser Benutzerkonto ist geschlossen.
```

Comment:
Diese**s**

Reply:
Ouch, another basic article mistake. Thanks, I also changed to "**was** closed" in English.